backend/apps/subject/management/commands/import_utest_subjects.py

```
@@ -0,0 +1,461 @@
+            name = (row.get('user_name') or '').strip()
+            if id_card:
+                id_card_name_freq[id_card][name] += 1
+
+        groups = {}
+
+        for row in rows:
+            id_card = (row.get('id_card') or '').strip()
+            phone = norm_phone(row.get('phone') or '')
+            name = (row.get('user_name') or '').strip()
+
+            # 优先级 1：身份证
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)